### DISCUSSION TITLE
Whisper hallucinated

### DISCUSSION THREAD

**Discussion by ArthurPeabody** [ggml-org/whisper.cpp#1199] [General]
I transcribed a radio show that usually transcribes easily.  Whisper found about 50 words that don't exist in the middle of the episode.  I listened to the channels separately, could hear nothing.  When I pointed Whisper at that 30 seconds it found nothing.  I don't mind.  It seems weird.  More often it loses track when there's music or other non-verbal content and stops transcribing, records everything as music or noise.

**Comment by ulatekh** [comment]
Whisper tends to hallucinate in silence or during non-speech noise. This is true of stock Whisper, i.e. it's not specific to whisper.cpp.
